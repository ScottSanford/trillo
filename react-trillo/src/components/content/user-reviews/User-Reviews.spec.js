import React from 'react'
import { shallow } from 'enzyme'
import UserReviews from './User-Reviews'

describe('UserReviews Component', () => {

	const minProps = {
		reviews: []
	}

	it('should match snapshot', () => {
		const wrapper = shallow(<UserReviews {...minProps} />)
		expect(wrapper).toMatchSnapshot()
	})
})