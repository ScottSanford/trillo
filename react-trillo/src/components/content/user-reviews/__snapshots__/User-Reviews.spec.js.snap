// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserReviews Component should match snapshot 1`] = `
<div
  className="user-reviews"
>
  <button
    className="btn-inline"
  >
    Show all 
    <span>
      â†’
    </span>
  </button>
</div>
`;
